Source/Core/DolphinWX/Config/AdvancedConfigPane.cpp

```diff
@@ -36,6 +36,10 @@ void AdvancedConfigPane::InitializeGUI()
 
   m_custom_rtc_checkbox = new wxCheckBox(this, wxID_ANY, _("Enable Custom RTC"));
   m_custom_rtc_date_picker = new wxDatePickerCtrl(this, wxID_ANY);
+  // The Gamecube allows setting a year between 2000 and 2099 in its native settings.
```

Comment:
I know this is really, really minor, but could you change "Gamecube" to "GameCube"? And by "native settings" do you mean the main menu / IPL?

Also, wouldn't it make more sense to mention the Wii date range first, then the GC range, and explain that this is why we set the range to 2000-2099 instead of 2000-2035?